doc/METADATA.ini

```diff
@@ -679,6 +679,43 @@ type= enum
 	RFC2822
 description= Support of some standards for a date specification.
 
+[check/permission]
+status= idea
+usedby/plugin= path
+type= enum
+description= A check to see if a certain user has correct permissions on a file. Permissions are
+	either r (read) w (write) x (execute) and any combination of them.
+	 The respective user which must have these permissions is declared in check/permission/user.
+example = "rwx ; rw ; r ; x ; w ; wx"
+
+[check/permission/user]
+status= idea
+usedby/plugin= path
+type= string
+	empty
+description= Used be check/permission to see if the given user has the correct permissions. If empty
+	it takes the current user.
```

Comment:
Can it be used without check/permission? Maybe assume rwx?

Reply:
Yes but I think it would lead to more errors since for example `x` is rarely used and attached to a file. 
For a default value I would suggest only `r`.

Still I think it's the best to let the user decide which permissions should be possible as it is very different from file to file. Maybe when we gather more data it is clear?